winml/lib/Api/LearningModelBinding.cpp

```diff
@@ -215,98 +218,103 @@ void LearningModelBinding::Split(
   throw hresult_not_implemented();
 }
 
-onnxruntime::IOBinding& LearningModelBinding::BindingCollection() {
-  return *m_lotusBinding;
+_winmla::IIOBinding* LearningModelBinding::BindingCollection() {
+    _winmla::IIOBinding* p;
+    m_lotusBinding.copy_to(&p);
+    return p;
```

Comment:
is this a copy or an add-ref?

Reply:
copy_to bumps the ref, so it is returning a strong ref.  the caller needs to handle it.    in this case there is one caller, in learningmodelsession.cpp line #206

---
In reply to: [347042063](https://github.com/microsoft/onnxruntime/pull/2406#discussion_r347042063) [](ancestors = 347042063)